### PR TITLE
misc: fix spelling

### PR DESCRIPTION
**Pull request by jbampton**
### Description

This PR fixes some typos

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [X] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [X] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [X] Trivial

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Review by DaanHoogland — APPROVED** [review]
text and code lgtm

File: server/src/test/java/com/cloud/network/vpc/NetworkACLServiceImplTest.java [hunk 1]
```diff
@@ -1183,7 +1183,7 @@ public void moveRuleToTheBottomTest() {
     }
 
     @Test(expected = InvalidParameterValueException.class)
-    public void moveRuleBetweenAclRulesTestThereIsSpaceBetweenPreviousRuleAndNextRuleToAccomodateTheNewRuleWithOtherruleColliding() {
+    public void moveRuleBetweenAclRulesTestThereIsSpaceBetweenPreviousRuleAndNextRuleToAccommodateTheNewRuleWithOtherruleColliding() {
```
Comment (stephankruggg) [inline server/src/test/java/com/cloud/network/vpc/NetworkACLServiceImplTest.java:1186]:
```suggestion
    public void moveRuleBetweenAclRulesTestThereIsSpaceBetweenPreviousRuleAndNextRuleToAccommodateTheNewRuleWithOtherRuleColliding() {
```

**Review by stephankruggg — APPROVED** [review]
CLGTM

File: server/src/test/java/com/cloud/network/vpc/NetworkACLServiceImplTest.java (hunk 1 above)
Comment (DaanHoogland) [reply server/src/test/java/com/cloud/network/vpc/NetworkACLServiceImplTest.java:1186]: guess I'm blind but what changed in your suggestion @stephankruggg ?

Comment (stephankruggg) [reply server/src/test/java/com/cloud/network/vpc/NetworkACLServiceImplTest.java:1186]: `WithOtherrule` -> `WithOtherRule`

**Comment by shwstppr** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@shwstppr I guess we can add @stephankruggg 's sugestion, can we?

**Review by shwstppr — APPROVED** [review]
LGTM

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
As there are fixes in DB sql code I'll run regression tests

**Comment by DaanHoogland** [comment]
@blueorangutan LLtest matrix

**Comment by vishesh92** [comment]
Changes in schema shouldn't be done for versions which have been released. This can cause issues in direct migration from an old version to the latest version. To fix values in database, I would recommend adding migrations in the latest schema files.

**Comment by vishesh92** [comment]
> Changes in schema shouldn't be done for versions which have been released. This can cause issues in direct migration from an old version to the latest version. To fix values in database, I would recommend adding migrations in the latest schema files.

The specific changes I am concerned about are:
* `vm.destroy.forcestop` - Direct migration from v2 to v4.19+ will cause issues. Otherwise it shouldn't be a problem.
* Guest OS mappings change. I am not sure how they work.

**Comment by DaanHoogland** [comment]
> > Changes in schema shouldn't be done for versions which have been released. This can cause issues in direct migration from an old version to the latest version. To fix values in database, I would recommend adding migrations in the latest schema files.
> 
> The specific changes I am concerned about are:
> 
>     * `vm.destroy.forcestop` - Direct migration from v2 to v4.19+ will cause issues. Otherwise it shouldn't be a problem.

I am not worried about those. People upgrading from before 4.0 will certainly need a more step upgrade to facilitate newer os and java versions along the way. Let's ignore ?!?

> 
>     * Guest OS mappings change. I am not sure how they work.

Can you point to those, please?

**Comment by vishesh92** [comment]
> > > Changes in schema shouldn't be done for versions which have been released. This can cause issues in direct migration from an old version to the latest version. To fix values in database, I would recommend adding migrations in the latest schema files.
> > 
> > 
> > The specific changes I am concerned about are:
> > ```
> > * `vm.destroy.forcestop` - Direct migration from v2 to v4.19+ will cause issues. Otherwise it shouldn't be a problem.
> > ```
> 
> I am not worried about those. People upgrading from before 4.0 will certainly need a more step upgrade to facilitate newer os and java versions along the way. Let's ignore ?!?
> 
> > ```
> > * Guest OS mappings change. I am not sure how they work.
> > ```
> 
> Can you point to those, please?

Search for `Bussiness`. It's for a very old version of `Microsoft Small Bussiness Server 2003`. So, not really sure about this either.

https://github.com/apache/cloudstack/pull/7549/files#diff-6ef445d28418f676c6c216081b90ec799f0b3b19536fccb4aee9477b2ae08b0eL150-R150

https://github.com/apache/cloudstack/pull/7549/files#diff-1f87f75156b9f12669a12a142ffc7fbe5198b1acf0b29201e24b83cc84ce2711L649-R649

https://github.com/apache/cloudstack/pull/7549/files#diff-bd11906a386daf528a2192d6e74836e4d4f4b8284b5e1ac64a5c1ce2b460195dL110-R110

**Comment by DaanHoogland** [comment]
> > > > Changes in schema shouldn't be done for versions which have been released. This can cause issues in direct migration from an old version to the latest version. To fix values in database, I would recommend adding migrations in the latest schema files.
> > > 
> > > 
> > > The specific changes I am concerned about are:
> > > ```
> > > * `vm.destroy.forcestop` - Direct migration from v2 to v4.19+ will cause issues. Otherwise it shouldn't be a problem.
> > > ```
> > 
> > 
> > I am not worried about those. People upgrading from before 4.0 will certainly need a more step upgrade to facilitate newer os and java versions along the way. Let's ignore ?!?
> > > ```
> > > * Guest OS mappings change. I am not sure how they work.
> > > ```
> > 
> > 
> > Can you point to those, please?
> 
> Search for `Bussiness`. It's for a very old version of `Microsoft Small Bussiness Server 2003`. So, not really sure about this either.
> 
> https://github.com/apache/cloudstack/pull/7549/files#diff-6ef445d28418f676c6c216081b90ec799f0b3b19536fccb4aee9477b2ae08b0eL150-R150
> 
> https://github.com/apache/cloudstack/pull/7549/files#diff-1f87f75156b9f12669a12a142ffc7fbe5198b1acf0b29201e24b83cc84ce2711L649-R649
> 
> https://github.com/apache/cloudstack/pull/7549/files#diff-bd11906a386daf528a2192d6e74836e4d4f4b8284b5e1ac64a5c1ce2b460195dL110-R110

That guestos is not inserted in any 4.0 or up install, I think we can pass that one as well, @vishesh92

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Pull request merged**